glotaran/builtin/models/kinetic_spectrum/kinetic_spectrum_matrix.py

```diff
@@ -8,9 +8,9 @@
 from .spectral_irf import IrfGaussianCoherentArtifact
 
 
-def kinetic_spectrum_matrix(dataset_descriptor=None, axis=None, index=None, irf=None):
+def kinetic_spectrum_matrix(dataset_descriptor=None, axis=None, index=None, extra=None):
 
-    clp_label, matrix = kinetic_image_matrix(dataset_descriptor, axis, index, irf)
```

Comment:
what was irf used for before, because irf was inside dataset_descriptor right?

Reply:
This as just a placeholder and never used.